.github/workflows/create-backport-trackers.yml

```
@@ -36,10 +36,11 @@ jobs:
     runs-on: ubuntu-latest
     if: github.ref == 'refs/heads/main'
     steps:
-      - uses: Bhacaz/checkout-files@e3e34e7daef91a5f237485bb88a260aee4be29dd
+      - uses: actions/checkout@v4
         with:
-          files: src/script/backport-create-issue src/script/requirements.backport-create-issue.txt
-      - uses: actions/setup-python@v4
+          sparse-checkout: src/script/backport-create-issue src/script/requirements.backport-create-issue.txt
```

If it works, great! However, I see in the docs:
```
    # Do a sparse checkout on given patterns. Each pattern should be separated with
    # new lines.
    # Default: null
    sparse-checkout: ''
```

So I don't know if we should also change the line to something like:
```suggestion
          sparse-checkout: |
            src/script/backport-create-issue
            src/script/requirements.backport-create-issue.txt
```

---

I can change that.  I don't think it matters to the yaml, but it does parse slightly differently (in both cases it's a list, but in the | case each string has an appended newline).